blog/2023-02-09-hassgetstate.md

```diff
@@ -0,0 +1,46 @@
+---
+author: Michael Hansen
+authorURL: https://twitter.com/rhasspy
+authorImageURL: /img/profile/mike_hansen.png
+authorTwitter: rhasspy
+title: The HassGetState intent
+---
+
+We've added a new [built-in intent](https://developers.home-assistant.io/docs/intent_builtin/): `HassGetState`
```

Comment:
```suggestion
We've added a new [built-in intent](/docs/intent_builtin/): `HassGetState`
```

Reply:
Could we link to the actual intent title hashtag?

Reply:
Seems like the HTML produced is just an `h3` with no `id`